protocol/messenger_communities.go

```diff
@@ -507,30 +506,8 @@ func (m *Messenger) CuratedCommunities() (*communities.KnownCommunitiesResponse,
 	// Revert code to https://github.com/status-im/status-go/blob/e6a3f63ec7f2fa691878ed35f921413dc8acfc66/protocol/messenger_communities.go#L211-L226 once the curated communities contract is deployed to mainnet
 
 	chainID := uint64(420) // Optimism Goerli
-	sDB, err := accounts.NewDB(m.database)
-	if err != nil {
-		return nil, err
-	}
-	nodeConfig, err := sDB.GetNodeConfig()
```

Comment:
Shall we add a todo in this getNodeConfig function to update and use network manager or this unexpected empty rpc url will be come back?

Reply:
added comment & removed method